### PR TITLE
Fix set "picked up" not showing in ready

### PR DESCRIPTION
**Pull request by jknapp25**
Option to set picked up was not surfacing in READY fulfillment state. I think this was just an oversight. At first we based off of order state, then fulfillment Received, should have been received/ready.

### CODE REVIEW

File: packages/sdk/src/types.ts [hunk 1]
```diff
@@ -416,7 +428,7 @@ export type OrderFilter = {
 export type OrderFulfillment = {
   __typename?: 'OrderFulfillment';
   carrier?: Maybe<Scalars['String']>;
-  state: Scalars['String'];
+  state: FulfillmentState;
```
Comment (samkotlove) [inline packages/sdk/src/types.ts:431]: Aren't these autogenerated?

Comment (jknapp25) [reply packages/sdk/src/types.ts:431]: I believe the types in `dist` get generated from src

Comment (marisbest2) [reply packages/sdk/src/types.ts:431]: I dont think we have type generation yet

**Comment by marisbest2** [comment]
FWIW I much prefer Typescript string literals to enums. Enums are much harder to read than strings, require being imported, are javascript objects under the hood, among other reasons

File: packages/sdk/src/types.ts (hunk 1 above)
Comment (captDaylight) [reply packages/sdk/src/types.ts:431]: correct, we don't, yet

**Review by marisbest2 — APPROVED** [review]

**Pull request merged**